modules/user-guide/pages/installing-vale-cli.adoc

```diff
@@ -1,49 +1,60 @@
 // Metadata for Antora
-:navtitle: Installing Vale by cloning the *{repository-name}* repo
+:navtitle: Installing Vale
 :keywords: vale
-:page-aliases: end-user-guide:using-vale-cli.adoc
-:description: Describes how to install and configure the Vale CLI
+:description: Describes how to install from the latest release and configure the Vale CLI
+:page-aliases: end-user-guide:using-vale-cli.adoc, installing-vale-cli-from-zip.adoc
 // End of metadata for Antora
-
+// Metadata for Modular Docs
 :context: assembly_getting-started-with-vale
 :_module-type: PROCEDURE
-[id="proc_using-vale-cli-on-a-local-environment_{context}"]
-= Installing and configuring Vale by cloning the *{repository-name}* repo
-
-To use `vale` CLI on any project in your local environment, install the `vale` CLI and register the *{repository-name}* Vale configuration as the default.
-
-NOTE: Alternatively, if you do not intend to contribute to the *{repository-name}* project, you can configure the `RedHat` Vale style by downloading the latest release from GitHub. See xref:installing-vale-from-zip.adoc[]
-
-.Prerequisites
+// End of metadata for Modular Docs
+[id="proc_using-vale-cli_{context}"]
+= Installing Vale with the `RedHat` package
```

Comment:
At this point, will the user be familiar with the concept of 'packages'. If not, perhaps the following might increase findability:

Suggestion: 'Installing Vale with the preconfigured Red Hat style ruleset package'